logger/logger.go

```diff
@@ -23,27 +27,44 @@ type Logger interface {
 	Error(string)
 }
 
+var logLevels = map[string]level.Option{
+	"debug": level.AllowDebug(),
+	"info":  level.AllowInfo(),
+	"warn":  level.AllowWarn(),
+	"error": level.AllowError(),
+}
+
 var _ Logger = (*logger)(nil)
 
 type logger struct {
 	kitLogger log.Logger
 }
 
 // New returns wrapped go kit logger.
-func New(out io.Writer) Logger {
+func New(out io.Writer, logLevel string) Logger {
 	l := log.NewJSONLogger(log.NewSyncWriter(out))
+	if allowedLevel, ok := logLevels[strings.ToLower(logLevel)]; ok {
+		l = level.NewFilter(l, allowedLevel)
+	} else {
+		l = level.NewFilter(l, level.AllowError())
```

Comment:
Please return error if invalid log level is provided.

Reply:
I've started returning an error and no longer default it to error if there is bad input.

Please let me know your thoughts on how I chose to handle it for now.
https://github.com/mainflux/mainflux/blob/2ffb2bd3ba384c7e9451fdb30862d9cbb0d0511f/cmd/normalizer/main.go#L95-L100
It got a little funky since there is no logger created yet at this point. 

So I think a few options are have each component create a logger and then update it with the appropriate logging level. Default if it is bad input or the above here. 
Thoughts?